apify-api/openapi/paths/request-queues/request-queues@{queueId}@requests@unlock.yaml

```diff
@@ -0,0 +1,55 @@
+post:
+  tags:
+    - Storage/Request queues/Requests locks
+  summary: Unlock requests
+  description: |
+    Unlocks requests in the queue for the client. The requests are unlocked if were locked by the same client key or if locked from the same Actor run.
+  operationId: requestQueue_requests_unlock_post
+  parameters:
+    - name: queueId
+      in: path
+      description: Queue ID or `username~queue-name`.
+      required: true
+      style: simple
+      schema:
+        type: string
+        example: WkzbQMuFYuamGv3YF
+    - name: clientKey
+      in: query
+      description: |
+        A unique identifier of the client accessing the request queue. It must
+        be a string between 1 and 32 characters long
+      style: form
+      explode: true
```

Comment:
Um, it's just a simple string query parameter, why `style: form` and `explode: true`?

Reply:
Honestly not sure, I just reused this param from the current rq endpoint see https://github.com/apify/apify-docs/blob/d676859747b5775d3562d88a518d8a5fc1b35610/apify-api/openapi/paths/request-queues/request-queues@%7BqueueId%7D@requests@batch.yaml#L153